django_select2/forms.py

```diff
@@ -98,12 +98,24 @@ def _get_media(self):
         .. Note:: For more information visit
             https://docs.djangoproject.com/en/1.8/topics/forms/media/#media-as-a-dynamic-property
         """
+        lang = get_language()
         try:
-            # get_language() will always return a lower case language code, where some files are named upper case.
-            i = [x.lower() for x in settings.SELECT2_I18N_AVAILABLE_LANGUAGES].index(get_language())
-            i18n_file = ('%s/%s.js' % (settings.SELECT2_I18N_PATH, settings.SELECT2_I18N_AVAILABLE_LANGUAGES[i]), )
-        except ValueError:
-            i18n_file = ()
+            from django.contrib.admin.widgets import SELECT2_TRANSLATIONS
+            i18n_name = SELECT2_TRANSLATIONS.get(lang)
+            if i18n_name not in settings.SELECT2_I18N_AVAILABLE_LANGUAGES:
+                i18n_name = None
+        except ImportError:
+            # TODO: select2 widget feature needs to be backported into Django 1.11
+            i18n_name = None
+            try:
+                i = [x.lower() for x in settings.SELECT2_I18N_AVAILABLE_LANGUAGES].index(lang)
+                i18n_name = settings.SELECT2_I18N_AVAILABLE_LANGUAGES[i]
+            except ValueError:
+                pass
+
+        i18n_file = (('%s/%s.js' % (settings.SELECT2_I18N_PATH, i18n_name),)
+                     if i18n_name in settings.SELECT2_I18N_AVAILABLE_LANGUAGES else ())
```

Comment:
I don't know whether this check is necessary, I guess currently it allow user to configure `SELECT2_I18N_AVAILABLE_LANGUAGES` ( for example, restrict i18n language allowed? I don't know). If not, it can be changed to 
```python 
if i18n_name else ()
```